### PR TITLE
[WIP] Announcement for Fine RC1

### PR DESCRIPTION
**Pull request by JPrause**

### CODE REVIEW

**Comment by JPrause** [comment]
@cybette please review. I used the format you made in the Fine Beta release.

File: site/_posts/2017-04-17-Announcing-Fine-RC1-Release.html.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+On behalf of the ManageIQ team, we're pleased to announce Release Candidate 1 for ManageIQ Fine!
+
+Fine is the next milestone release for ManageIQ cloud and virtualization management platform. With each release, ManageIQ gets more robust and feature complete across providers. In this release, a lot of attention was given to Ansible. However, other providers, REST API, as well as performance also benefit in this release.
+
+* Added **"Ansible Playbook"** to the list of catalog item types.
+* Automate Model: Add Amazon block storage automation models.
+* Adding new report and widgets for Containers and Add option for container performance reports.
+* Add Alert Definitions (MiqAlert) bulk edits support.
+* Middleware: Hawkular Allow adding datawarehouse provider with a port other than 80.
```
Comment (cybette) [inline site/_posts/2017-04-17-Announcing-Fine-RC1-Release.html.md]: This list should include the deltas between the beta and RC1.

The rest LGTM! :+1:

Comment (JPrause) [inline site/_posts/2017-04-17-Announcing-Fine-RC1-Release.html.md]: Thanks @cybette I'll get that info updated.

**Comment by JPrause** [comment]
@cybette got the okay from QE on smoke testing. Please merge when possible.

**Comment by cybette** [comment]
Awaiting feedback whether this is a beta release or can be considered RC.

**Pull request closed without merging**